stub/src/test/java/io/grpc/stub/ClientCallsTest.java

```diff
@@ -65,6 +65,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicReference;
 import org.junit.After;
+import org.junit.Assert;
```

Comment:
nit: noticed there's already a static import of `Assert.fail`, could you please replace `Assert.fail()` with `fail()` and remove this import?

Reply:
done